### PR TITLE
JS: Add textbox module

### PR DESCRIPTION
**Pull request by WillyJL**
# What's new

- JS API for textbox module
- #3586

# Verification 

- Apps > Scripts > textbox.js

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

**Review by nminaylov — DISMISSED** [review]

File: applications/system/js_app/modules/js_textbox.c [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    mjs_return(mjs, MJS_UNDEFINED);
+}
+
+static void* js_textbox_create(struct mjs* mjs, mjs_val_t* object) {
+    JsTextboxInst* textbox = malloc(sizeof(JsTextboxInst));
+    mjs_val_t textbox_obj = mjs_mk_object(mjs);
+    mjs_set(mjs, textbox_obj, INST_PROP_NAME, ~0, mjs_mk_foreign(mjs, textbox));
+    mjs_set(mjs, textbox_obj, "setConfig", ~0, MJS_MK_FN(js_textbox_set_config));
+    mjs_set(mjs, textbox_obj, "addText", ~0, MJS_MK_FN(js_textbox_add_text));
+    mjs_set(mjs, textbox_obj, "emptyText", ~0, MJS_MK_FN(js_textbox_empty_text));
```
Comment (anna-oake) [inline applications/system/js_app/modules/js_textbox.c]: `clearText()` maybe? or just `clear()` even

**Comment by WillyJL** [comment]
First thing I thought was to use view holder actually, but then noticed it's not exposed, glad to see it in public api )

File: applications/system/js_app/modules/js_textbox.c (hunk 1 above)
Comment (WillyJL) [reply applications/system/js_app/modules/js_textbox.c]: Since it's `addText()`, `clearText()` sounds more intuitive to me, but both work well enough

**Comment by nminaylov** [comment]
Yeah, we forgot to include it in API for some reason

**Review by skotopes — APPROVED** [review]

**Pull request merged**